blog/_posts/2016-12-xx-moredots.md

```diff
@@ -0,0 +1,456 @@
+---
+layout: post
+title:  More Dots: Syntactic Loop Fusion in Julia
+author: <a href="http://math.mit.edu/~stevenj">Steven G. Johnson</a>
+---
+
+After a [lengthy design process](https://github.com/JuliaLang/julia/issues/8450) and [preliminary foundations in Julia 0.5](http://julialang.org/blog/2016/10/julia-0.5-highlights#vectorized-function-calls), Julia 0.6 includes new facilities for writing code in the "vectorized"
+style (familiar from Matlab, Numpy, R, etcetera) while avoiding the
+overhead that this style of programming usually imposes: multiple
+vectorized operations can now be "fused" into a single loop, without
+allocating any extraneous temporary arrays.
+
+This is best illustrated with an example.  Suppose we have
```

Comment:
This post is fantastic. The more who read it through, the better. In the interest of hooking readers as soundly as possible from the get-go, augmenting this introductory example with a simple, stark performance "wow!" would be great. (I wager that an explicit runtime reduction illustration will better capture most readers' attention than the existing description of fusion into a single loop --- to have that description hit home as a "wow!", the reader need already appreciate most of the content of this post.) Thoughts? Best!